extensions/wikia/AdEngine3/src/slots.js

```diff
@@ -361,20 +355,10 @@ export default {
 	},
 
 	injectIncontentPlayer() {
-		const header = document.querySelectorAll('#mw-content-text > h2')[1];
-
-		if (!header || !isIncontentPlayerApplicable()) {
-			return;
-		}
-
-		const slotName = 'incontent_player';
-		const wrapper = document.createElement('div');
-
-		wrapper.id = 'INCONTENT_WRAPPER';
-		wrapper.innerHTML = '<div id="' + slotName + '" class="wikia-ad hide"></div>';
-		header.parentNode.insertBefore(wrapper, header);
+		const isApplicable = !context.get('custom.hasFeaturedVideo');
```

Comment:
Would you please add a check if the page type is article? I believe we forgot about this :(